lerobot/common/robot_devices/cameras/intelrealsense.py

```diff
@@ -0,0 +1,435 @@
+"""
+This file contains ultilities for recording frames from Intel Realsense cameras.
+"""
+
+import argparse
+import concurrent.futures
+import logging
+import shutil
+import threading
+import time
+import traceback
+from dataclasses import dataclass, replace
+from pathlib import Path
+from threading import Thread
+
+import cv2
+import numpy as np
+import pyrealsense2 as rs
```

Comment:
We should make sure that this is not executed when pyrealsense2 isn't installed (especially on MacOS)

Reply:
It won't be.